### DISCUSSION TITLE
Events not getting emitted in local blockchain but visible in Remix IDE

### DISCUSSION THREAD

**Discussion by arjunpathy** [ConsenSys-archive/truffle#6084] [General Q&A]
I am a newbie in blockchain development and currently learning about "Events".  My code emits events in Remix IDE but the same code in local with the same ganache, the receipts have empty logs. I want to capture and display the event in my UI to fix my local configuration. Could anyone possibly point out what I am missing? 
![Screenshot 2023-06-08 201528](https://github.com/trufflesuite/truffle/assets/21103179/00cb7b85-1428-4fe8-bb4d-24262e5d55cf)
![Screenshot 2023-06-08 201404](https://github.com/trufflesuite/truffle/assets/21103179/d268e366-32c2-446d-94cd-a59634937270)

**Comment by FlaviuWeb3** [comment]
Hi @arjunpathy could you please share a repository so we can troubleshoot this?
